components/ws-daemon/pkg/content/service.go

```diff
@@ -396,6 +396,9 @@ func (s *WorkspaceService) DisposeWorkspace(ctx context.Context, req *api.Dispos
 	return resp, nil
 }
 
+// channel to limit the number of concurrent backups and uploads.
+var backupWorkspaceLimiter = make(chan bool, 3)
```

Comment:
I think it would be better to make the number be able to control with config. WDYT?